tests/foreman/ui/test_contenthost.py

```diff
@@ -315,6 +315,46 @@ def test_positive_search_by_subscription_status(session, vm):
         assert values['table'][0]['Name'] == vm.hostname
 
 
+@pytest.mark.tier3
+def test_positive_toggle_subscription_status(session, vm):
+    """Register host into the system, assert subscription status valid,
+    toggle status off and on again using CLI and assert status is updated in web UI.
+
+    :id: b9343e6f-9354-46ef-873f-b63851d29043
+
+    :expectedresults: Subscription status changed on the CLI is reflected in the web UI.
+
+    :customerscenario: true
+
+    :BZ: 1836868
+
+    :CaseLevel: System
+
+    :CaseImportance: Medium
+    """
+    with session:
+        session.location.select('Default Location')
+        assert session.contenthost.search(vm.hostname)[0]['Name'] == vm.hostname
+        subscriptions = session.contenthost.read(vm.hostname, widget_names='subscriptions')[
+            'subscriptions'
+        ]
+        assert subscriptions['auto_attach'] == 'Yes'
+        # Toggle auto-attach status to No using CLI
+        vm.run('subscription-manager auto-attach --disable && subscription-manager refresh')
```

Comment:
Is using vm.run the only way to disable auto-attach and refresh sub-man? do we have anything in airgun/nailgun/robottelo that can do this ?

Reply:
Hi @ColeHiggins2 

Airgun is for communication with web UI
Nailgun is for communication with API

In this test it has to be CLI. I could not find 

There is an subscription_auto_attach function here:
https://github.com/SatelliteQE/robottelo/blob/ed7d3950f708f7bf33cbe0ff89a82c06466ec94d/robottelo/cli/host.py#L405

but that command does not support --true and --false options, like the activation-key command does for auto-attach.

But even it it did, I would still need the refresh step.